### PR TITLE
Adding 'Resoto' to the Software Development - IDE & Tools category.md

### PR DESCRIPTION
**Pull request by scapecast**
Proposing to add Resoto, our open source cloud asset inventory, to the development / IDE category.

Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software) must be added to `non-free.md` and have the license set to `⊘ Proprietary`
- [x] Additions are inserted preserving alphabetical order.
- [x] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [awesome-analytics](https://github.com/onurakpolat/awesome-analytics), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/).
- [x] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [x] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [x] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Misc/Other`.
- [x] Any software project you are adding to the list is actively maintained.
- [x] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1811,6 +1811,7 @@ An [integrated development environment (IDE)](https://en.wikipedia.org/wiki/Inte
 - [ML Workspace](https://github.com/ml-tooling/ml-workspace) - All-in-one web-based IDE for machine learning and data science. `Apache-2.0` `Docker`
 - [Motor Admin](https://www.getmotoradmin.com/) - No-code admin panel and business intelligence software - search, create, update, and delete data entries, create custom actions, and build reports. ([Demo](https://motor-admin.herokuapp.com/demo/), [Source Code](https://github.com/motor-admin/motor-admin)) `AGPL-3.0` `Ruby`
 - [RStudio Server](https://www.rstudio.com/products/rstudio/#Server) - Web browser based IDE for R. ([Source Code](https://github.com/rstudio/rstudio)) `AGPL-3.0` `Java/C++`
+- [Resoto] (https://resoto.com) - Resoto is an open source cloud asset inventory. Write code that reacts to changes in your cloud infrastructure. Resoto helps you search infrastructure, generate reports, and automate tasks. ([Source Code](https://github.com/someengineering/resoto)) `Apache-2.0` `Python/Docker`
```
Comment (KieranRobson) [inline README.md:1814]: - Please remove the extra space between [Resoto] and (https://resoto.com)

> https://resoto.com/docs#what-is-resoto

-  Please change the description to  `Create indexes resources, captures dependencies, and maps out your infrastructure in an intuitive [graph](https://resoto.com/docs/concepts/graph). The graph contains metrics for each [resource](https://resoto.com/docs/reference/data-models).`

**Review by KieranRobson — CHANGES_REQUESTED** [review]
Thanks for the contrbution!

A couple changes to be made

**Review by KieranRobson — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -1811,6 +1811,7 @@ An [integrated development environment (IDE)](https://en.wikipedia.org/wiki/Inte
 - [ML Workspace](https://github.com/ml-tooling/ml-workspace) - All-in-one web-based IDE for machine learning and data science. `Apache-2.0` `Docker`
 - [Motor Admin](https://www.getmotoradmin.com/) - No-code admin panel and business intelligence software - search, create, update, and delete data entries, create custom actions, and build reports. ([Demo](https://motor-admin.herokuapp.com/demo/), [Source Code](https://github.com/motor-admin/motor-admin)) `AGPL-3.0` `Ruby`
 - [RStudio Server](https://www.rstudio.com/products/rstudio/#Server) - Web browser based IDE for R. ([Source Code](https://github.com/rstudio/rstudio)) `AGPL-3.0` `Java/C++`
+- [Resoto](https://resoto.com) - Resoto creates an inventory of your cloud, indexes resources, captures dependencies, and maps out your infrastructure in an intuitive [graph](https://resoto.com/docs/concepts/graph). The graph contains metrics for each [resource](https://resoto.com/docs/reference/data-models) ([Source Code](https://github.com/someengineering/resoto)) `Apache-2.0` `Python/Docker`
```
Comment (nodiscc) [inline README.md:1814]: - Missing `.` at the end of the description
- Please remove inline links to graph/resource.

**Review by nodiscc — CHANGES_REQUESTED** [review]

**Comment by nodiscc** [comment]
No reply for > a month, closing.

**Pull request closed without merging**